azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/implementation/ApplicationGatewayListenerImpl.java

```diff
@@ -193,6 +193,21 @@ public ApplicationGatewayListenerImpl withSslCertificate(String name) {
         return this;
     }
 
+    @Override
+    public ApplicationGatewayListenerImpl withKeyVaultSecretId(String keyVaultSecretId) {
+        return withKeyVaultSecretId(keyVaultSecretId, null);
```

Comment:
Why provide null value?

Reply:
`null` used for random name, like other ssl certificate function